pavelib/assets.py

```diff
@@ -636,6 +636,30 @@ def collect_assets(systems, settings):
         print("\t\tFinished collecting {} assets.".format(sys))
 
 
+def execute_compile_sass(args):
+    """
+    Construct django management command compile_sass (defined in theming app) and execute it.
+    Args:
+        args: command line argument passed via update_assets command
+    """
+    for sys in args.system:
+        options = ""
+        options += " --themes " + " ".join(args.themes) if args.themes else ""
+        options += " --themes-dir=" + args.themes_dir if args.themes_dir else ""
+        options += " --debug" if args.debug else ""
+
+        sh(
```

Comment:
@andy-armstrong the reason behind calling django management command is that we need to access django settings (and in future, may be, db data to fetch SiteConfiguration). I am calling back paver command to allow backward compatibility (of compile_sass) and avoid code duplication.